### PR TITLE
made edits as per ticket 94

### PR DESCRIPTION
**Pull request by CaitlinBate**
edits from https://github.com/ocean-tracking-network/node-manager-training/issues/94

### CODE REVIEW

File: _episodes/06_auth_notebook.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ questions:
 objectives:
 - "Understand how to use the `AUTH - Create and Update` notebook to maintain your database credentials file"
 keypoints:
-- "Node-members cannot access the database, you are the liason"
+- "Node-members cannot access the database, you are the liasion"
```
Comment (jackVanish) [inline _episodes/06_auth_notebook.md]: Hate to be a pedant but it is 'liaison'.

File: _episodes/10_Detections.md [hunk 2]
```diff
@@ -135,15 +136,18 @@ This section will be common for most Nodebooks: it is a cell at the top of the n
...
+- vrl_dir: Within quotation marks, paste the path to a folder containing the VRLs or VDATs you would like to convert. Example: `vrl_dir = r"/path/to/vrl_files/"`
+- export_path: Within quotation marks, paste the path to the folder to which you would like the CSV exported. Example: `export_path = r"/path/to/csv_export/"`
 
 Run this cell. There is no output.
 
 ### Check vdat exists
 
 The Nodebook will indicate the file had passed quality control by adding a ✔️**green checkmark** and printing the vdat version.
 
+**NOTE:** Older versions of VDAT may have unintended consequences when converting newer files (like Open Protocol-enabled Innovasea receivers), and should not be used. Versions > vdat-9.3.0-20240207-74ad8e-release are safe to process Open Protocol data.
```
Comment (jackVanish) [inline _episodes/10_Detections.md:148]: I would make this "versions newer than" rather than "versions >".

File: _episodes/11_movers_notebooks.md [hunk 3]
```diff
@@ -301,18 +301,18 @@ Cell two requires input from you. This information will be used to get the raw e
...
     * Paste a filepath to the relevant CSV file. The filepath will be added between the provided quotation marks.
 1. `table_name = 'c_events_YYYY_mm'`
-	* Within the quotes, please add your custom table suffix. We recommend using `year_month` or similar, to indicate the most-recently downloaded instrument.
+	* Within the quotes, please add your custom table suffix. We recommend using the year and month (i.e, `YYYY_MM`) or similar, to indicate the most-recently downloaded instrument.
 1. `schema = 'collectioncode'`
 	* Please edit to include the relevant project code, in lowercase, between the quotes.
 
 There are also some optional inputs:
-- `file_encoding`: The file_encoding: ISO-8859-1 in the event export. The  default encoding used in VUE's event export
+- `file_encoding`: How the file is encoded. ISO-8859-1 in the event export. The default encoding used in VUE's event export
```
Comment (jackVanish) [inline _episodes/11_movers_notebooks.md]: This sentence remains unclear. Is it saying ISO-8859-1 is the default encoding in VUE's event export?

**Review by jackVanish — CHANGES_REQUESTED** [review]
A few small edits for clarity but beyond that this seems fine to merge. Once those changes are made I'm happy to fold it in.

**Comment by CaitlinBate** [comment]
ready when u are!

**Comment by jackVanish** [comment]
Looks good to me!

**Pull request merged**